mllib-local/pom.xml

```diff
@@ -81,6 +88,34 @@
         </dependency>
       </dependencies>
     </profile>
+    <profile>
+      <id>vectorized</id>
+      <properties>
+        <extra.source.dir>src/vectorized/java</extra.source.dir>
```

Comment:
Just some nits now: I wonder if we should call this profile something more like `jvm-vectorized` to be more specific? but it doesn't matter much

Reply:
Let me do the rename.